Add input clear button background instruction (#126)
eclipsesource/jsonforms2-website: src/pages/docs/controls.mdx

In the following example each element will be labeled with its name instead of its message because we provide the name prop.

<ExampleWithArrayLabel />

> 

Reviewer: ```suggestion
## Theming
```

> ### Use custom theme to change input clear background

Reviewer: ```suggestion
### Customize 'clear input button' background
```

```
<ThemeProvider
  theme={{
    jsonforms: { input: { delete: { background: '#f44336' }}}
  }}
>
  <JsonForms
    ...
  />
</ThemeProvider>
```

> If you want to customize the background of the input clear button (the one you can see, if you hover of the input field), you can need to wrap you `JsonForms` component in an `ThemeProvider` component and provide your desired background color to the theme.

Reviewer: ```suggestion
The background of the the 'clear input button' (the one you can see when you hover of the input field) is by default your theme's `palette.background.default` color. If you want to customize the background of this button you can use the custom theme variable `jsonforms.input.delete.background`.
```
